functions/src/index.ts

```diff
@@ -0,0 +1,81 @@
+import admin from "firebase-admin";
```

Comment:
[note] This file was originally moved from `functions/index.js` + refactored to make better use of TypeScript/newer JS/TS features.